neo/Persistence/LevelDB/LevelDBStore.cs

```diff
@@ -34,9 +34,9 @@ public void Dispose()
             db.Dispose();
         }
 
-        public override byte[] Get(byte prefix, byte[] key)
+        public override byte[] Get(byte[] key)
         {
-            if (!db.TryGet(ReadOptions.Default, SliceBuilder.Begin(prefix).Add(key), out Slice slice))
+            if (!db.TryGet(ReadOptions.Default, key, out Slice slice))
```

Comment:
Very good Erik... in the end, it was just being merged here. Very good, simple and better performance :+1: